### PR TITLE
Adds json schema validator to fix issue #14

### PR DESCRIPTION
**Pull request by nyoungstudios**
Adds json schema validator to fix issue #14
Adds moment.js to dependencies
And changes the test-service to use local module for development

### CODE REVIEW

**Comment by nyoungstudios** [comment]
Only problem is that I couldn't figure out how to add a custom error message. So, currently it prints out all the valid timezone names if the provided timezone doesn't match the schema.

**Comment by edlea** [comment]
Rather than add a dependency on `moment-timezone` it might be simpler just to set the type of `timezone` as `string`

**Review by cedric-r-mycelium — APPROVED** [review]

**Comment by KillDozerX2** [comment]
@pgrzesik Can this be merged? I've got another [PR](https://github.com/nyoungstudios/serverless-local-schedule/pull/1) if you don't like this, can move it here directly

File: index.js [hunk 1]
```diff
@@ -121,6 +122,16 @@ function convertCrontabs() {
 class ServerlessLocalCrontabs {
   constructor(serverless, options) {
     this.serverless = serverless;
+
+    // Create schema for your properties. For reference use https://github.com/ajv-validator/ajv
+    this.serverless.configSchemaHandler.defineFunctionEventProperties('aws', 'schedule', {
```
Comment (pgrzesik) [inline index.js:127]: it should check if `this.serverless.configSchemaHandler` is defined first to avoid breaking for older Framework versions

**Review by pgrzesik — CHANGES_REQUESTED** [review]
looks good overall, but requires an adjustment to not. break for older Framework versions cc @KillDozerX2

File: index.js (hunk 1 above)
Comment (KillDozerX2) [reply index.js:127]: I will send PR directly then

**Comment by pgrzesik** [comment]
Close in favor of: https://github.com/serverless/serverless-local-schedule/pull/25

**Pull request closed without merging**